# HG changeset patch
# User Andrew John Hughes <gnu_andrew@member.fsf.org>
# Date 1454117106 0
#      Sat Jan 30 01:25:06 2016 +0000
# Node ID 130888a5c7134f73540323d45f64e2e7372c4a01
# Parent  e4660f6210fa312c8392ab2be79f85bb86673a85
PR1983: Support using the system installation of NSS with the SunEC provider

2016-01-29  Andrew John Hughes  <gnu.andrew@member.fsf.org>

	PR1983: Support using the system installation
	of NSS with the SunEC provider
	* Makefile.am:
	(check-ecc): Make argument dependent on whether
	the SunEC provider is enabled or not.

diff -r e4660f6210fa -r 130888a5c713 Makefile.am
--- a/Makefile.am	Sat Jan 30 01:15:16 2016 +0000
+++ b/Makefile.am	Sat Jan 30 01:25:06 2016 +0000
@@ -209,6 +209,12 @@
 TESTS_TO_RUN = jtreg $(addprefix check-,$(TEST_SUITES))
 endif
 
+if ENABLE_SUNEC
+ECC_RESULT = yes
+else
+ECC_RESULT = no
+endif
+
 # Target to ensure a patched OpenJDK tree containing Zero & Shark
 # and any overlays is available in $(abs_top_builddir)/openjdk
 OPENJDK_TREE = stamps/overlay.stamp
@@ -2367,7 +2373,7 @@
 
 stamps/check-ecc.stamp: stamps/ecccheck.stamp stamps/icedtea.stamp
 	if [ -e $(BUILD_SDK_DIR)/bin/java ] ; then \
-	  $(BUILD_SDK_DIR)/bin/java -cp $(ECC_CHECK_BUILD_DIR) TestEllipticCurveCryptoSupport no ; \
+	  $(BUILD_SDK_DIR)/bin/java -cp $(ECC_CHECK_BUILD_DIR) TestEllipticCurveCryptoSupport $(ECC_RESULT) ; \
 	fi
 	mkdir -p stamps
 	touch $@
